loadrunner_professional/manifest.json

```diff
@@ -0,0 +1,54 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "e6b5ab52-139d-4dde-a4ad-94fedeac7f29",
+  "app_id": "loadrunner-professional",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Send your LoadRunner Professional metrics from the scenario run, such as Vuser status and transaction response times, information about the scenario run, such as the start and stop time, duration, and included scripts, or both.",
```

Comment:
Sorry - looks like this is over the 120 character limit, please shorten. Thanks!